src/awkward/operations/ak_almost_equal.py

```diff
@@ -52,6 +52,29 @@ def almost_equal(
     # Dispatch
     yield left, right
 
+    return _impl(
+        left,
+        right,
+        rtol=rtol,
+        atol=atol,
+        dtype_exact=dtype_exact,
+        check_parameters=check_parameters,
+        check_regular=check_regular,
+    )
+
+
+def _impl(
+    left,
+    right,
+    rtol: float = 1e-5,
+    atol: float = 1e-8,
+    dtype_exact: bool = True,
+    check_parameters: bool = True,
+    check_regular: bool = True,
```

Comment:
```suggestion
    rtol: float,
    atol: float,
    dtype_exact: bool,
    check_parameters: bool,
    check_regular: bool,
```

Duplicating the user-facing default arguments with internal default arguments runs the risk of letting them diverge, since there would appear to be two sources of truth.

Reply:
Good point. I fixed these and used consistent parameter ordering.